src/components/settings/Settings.h

```diff
@@ -117,20 +121,10 @@ namespace Pinetime {
       uint8_t appMenu = 0;
       uint8_t settingsMenu = 0;
 
-      // There are 10 blocks of reserved flash to save settings
-      // to minimize wear, the recording is done in a rotating way by the 10 blocks
-      uint8_t settingsFlashBlock = 99; // default to indicate it needs to find the active block
-
-      static constexpr uint32_t settingsBaseAddr = 0x3F6000; // Flash Settings Location
-      static constexpr uint16_t settingsVersion = 0x0100;    // Flash Settings Version
+      lfs_file_t settingsFile;
```

Comment:
`settingsFile` doesn't need to be a member variable, it's only used in `LoadSettingsFromFile` and `SaveSettingsToFile` and in each of the those it's used to open, read/write, then close the file and a local variable would suffice.

Reply:
Thank you, I will review the recommendations.